lerobot/common/policies/act/modeling_act.py

```diff
@@ -487,23 +483,24 @@ def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, tuple[Tensor, Tenso
 
         # Camera observation features and positional embeddings.
         if self.config.image_features:
-            all_cam_features = []
-            all_cam_pos_embeds = []
+            all_tokens = []
+            all_pos_tokens = []
```

Comment:
tokens are supposed to be string/integer ; here it's more features. I would revert the naming